src/lib.rs

```diff
@@ -176,6 +176,10 @@ pub(crate) mod process_impl;
 #[cfg(feature = "native-mux")]
 pub(crate) mod native_mux_impl;
 
+#[cfg(docsrs)]
```

Comment:
I am afraid changing it to `doc` would cause it to disappear from doc.

From `Cargo.toml`:

```
[package.metadata.docs.rs]                    
# document all features                       
all-features = true                           
# defines the configuration attribute `docsrs`
rustdoc-args = ["--cfg", "docsrs"]            
```

Unless we also change the `rustdoc-args` specified in `Cargo.toml`.

Reply:
`cfg(doc)` should always be true whenever documentation is generated. It's not something we need to set ourselves.

Reply:
I didn't know `cfg(doc)` is set for documentation generation, now I have updated the code!